ui/app/src/components/shared/Tooltip.vue

```diff
@@ -0,0 +1,68 @@
+<script>
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  props: {
+    message: {
+      type: String,
+    },
+  },
+
+  data: function() {
+  	return {
+  		opened: false
```

Comment:
Dit auto-format it like that? could be as 1 line or have 2 spaces I think. @ryardley @59023g In general, I think we should have a default code style across the team so we don't restyle each other's code on every commit. It really makes commits look much bigger than they are and also makes it harder to review and easier to miss bugs.
What do you think about using prettier with default settings?

Reply:
I am a super prettier fanboy, I would love if we can use it. 

I think they do a thorough job of explaining the benefits for the team -> https://prettier.io/docs/en/why-prettier.html

Reply:
I know @ryardley had a linter in mind. Prettier sounds good to me I'll ping the group in Slack. We will find consensus